onnxruntime/core/optimizer/transpose_optimization/optimizer_api.h

```diff
@@ -480,6 +475,10 @@ struct OptimizeResult {
   bool graph_modified{false};
 };
 
+// see transpose_optimizer.h if you wish to provide extended handlers
```

Comment:
nit: now there are two files named transpose_optimizer.h - could one of them be renamed?